components/AnnoucementCard/AnnouncementCard.tsx

```diff
@@ -1,7 +1,7 @@
 
 import { styles } from "./styles"
-import React,{ View, Text }  from "react-native"
-import { Announcement } from "../../types/Annoucements"
+import React, { View, Text } from "react-native"
+import { Announcement } from "../../types/Annoucement"
```

Comment:
The file is named Annoucement. It appeared in a previous PR, if you could just change it's name to Announcement could be awesome !